editor.planx.uk/src/types.ts

```diff
@@ -36,6 +36,7 @@ export interface TeamSettings {
     url: string;
   };
   supportEmail?: string;
+  boundary?: string;
```

Comment:
nit: `boundaryURL`?

Reply:
i also considered this but settled on plain "boundary" after reviewing other jsonb key names in `teams` - "logo" for example also expects a URL but we keep a simple key name, "homepage" etc !

Reply:
That makes perfect sense, thanks for clarifying that 👍